Demo/Sources/Extensions/CMTime.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import CoreMedia
+import SwiftUI
+
+extension CMTime {
+    private static let shortFormatter: DateComponentsFormatter = {
+        let formatter = DateComponentsFormatter()
+        formatter.allowedUnits = [.second, .minute]
```

Comment:
Reordering for more natural reading: 

```suggestion
        formatter.allowedUnits = [.minute, .second]
```